### PR TITLE
ref(breadcrumbs): Adjust file structure

### PR DESCRIPTION
**Pull request by priscilawebdev**
renamed files `breadcrumbs/index.mdx` in `breadcrumbs.mdx` + removed unnecessary folder

### CODE REVIEW

File: src/platforms/javascript/common/errors/index.mdx [hunk 1]
```diff
@@ -1,30 +0,0 @@
----
-title: Enrich Event Data
```
Comment (rhcarvalho) [inline src/platforms/javascript/common/errors/index.mdx:2]: Is removing this file intentional? From the description I expected the moving of the two files (breadcrumbs-related) but not the deletion of this page about errors and the png image.

Comment (priscilawebdev) [reply src/platforms/javascript/common/errors/index.mdx]: no no this was a mistake... I just fixed it...thanks :)

Comment (rhcarvalho) [reply src/platforms/javascript/common/errors/index.mdx]: You're welcome ❤️

**Review by rhcarvalho — APPROVED** [review]
The change looks reasonable.

Notes:

- `toggle-time-btn.png` is apparently unused (only reference I found was in [/src/docs/product/error-monitoring/breadcrumbs/index.mdx](https://raw.githubusercontent.com/getsentry/sentry-docs/72dbb800137da5697aad0472a4b20e5ba010e499/src/docs/product/error-monitoring/breadcrumbs/index.mdx))
- The two breadcrumb pages look extremely similar, with a few differences that seem unnecessary. Tagging @PeloWriter for visibility on the duplication, but I'm assuming it is out-of-scope for this PR and already being worked on.

Thanks @priscilawebdev!

**Comment by PeloWriter** [comment]
@rhcarvalho - yes, those pages are going to be merged :) Thanks for noticing.

**Pull request merged**